### PR TITLE
interfaces/builtin: Add OVN to openvswitch interface

### PR DESCRIPTION
**Pull request by xavpaice**
Adds to the openvswitch interface the needed OVN pid files and sockets
for monitoring and updating OVN.

### CODE REVIEW

File: interfaces/builtin/openvswitch.go [hunk 1]
```diff
@@ -33,12 +33,32 @@ const openvswitchBaseDeclarationSlots = `
 // sockets needed by ovs-vsctl, ovs-ofctl and ovs-appctl commands. ovs-ofctl
```
Comment (bboozzoo) [inline interfaces/builtin/openvswitch.go:33]: Does the list of *ctl commands needs to be updated as well?

File: interfaces/builtin/openvswitch.go [hunk 2]
```diff
@@ -33,12 +33,32 @@ const openvswitchBaseDeclarationSlots = `
...
+/run/openvswitch/ovsdb-server.pid r,
+/run/ovn/ovn-northd.pid r,
+/run/ovn/ovnsb_db.pid r,
+/run/ovn/ovnsb_db.ctl rw,
+/run/ovn/ovnsb_db.sock rw,
+/run/ovn/ovnnb_db.pid r,
+/run/ovn/ovnnb_db.ctl rw,
+/run/ovn/ovnnb_db.sock rw,
+/run/openvswitch/ovsdb-server.*.ctl rw,
+/run/ovn/ovn-northd.*.ctl rw,
```
Comment (diddledani) [inline interfaces/builtin/openvswitch.go:61]: Note that `/var/run` and `/run` are usually the same location on a lot of distros. Are you sure you need the `/var`?

Comment (xavpaice) [reply interfaces/builtin/openvswitch.go]: thanks - I've removed the /var/run entries as they appear to be paths that are configurable in the application in any case.

File: interfaces/builtin/openvswitch.go (hunk 1 above)
Comment (xavpaice) [reply interfaces/builtin/openvswitch.go:33]: not for the use case I'm writing this change for, as it queries the socket directly.  See https://github.com/greenpau/ovsdb for the details.  The change is to support using https://github.com/greenpau/ovn_exporter for statistics gathering.

**Comment by Meulengracht** [comment]
@xavpaice is this still relevant? otherwise i'll close it soon

**Comment by ernestl** [comment]
Closing this due to lack of response from author

**Pull request closed without merging**